src/nycdb/datasets/marshal_evictions.yml

```diff
@@ -76,3 +79,21 @@ schema:
       MarshalLastName: text
       ResidentialCommercialInd: text
       ScheduleStatus: text
+  -
+    table_name: marshal_evictions_all
```

Comment:
Does this `table_name` seem appropriate? I wanted to just name this `marshal_evictions` but that created a weird conflict where the larger dataset name and this table name were identical. What do folks think?